core/vm/actor/version.hpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <type_traits>
```

Comment:
please remove, WithActorVersion subclass not checked anymore

Reply:
Missed it, thanks